pkg/lib/cockpit-components-listing.jsx

```diff
@@ -403,3 +405,35 @@ Listing.propTypes = {
     columnTitleClick: PropTypes.func,
     actions: PropTypes.node
 };
+
+/* Implements PF4 compatible listing with clicable rows
+ * Used in dialogs
+ *
+ * - rows {object}: with properties: name
+ * - onRowToggle {function}: toggle handler which takes row name as an arguement
+ * - idPrefix {string}
+ */
+export const SelectableListing = ({ rows, onRowToggle, idPrefix }) => {
+    return (
+        <fieldset className="list-group dialog-list-ct pf-c-select" id={idPrefix + "-selectable-listing"}>
+            {rows.map(row => {
+                return (
+                    <label className="pf-c-check pf-c-select__menu-item" htmlFor={idPrefix + "-" + row.name} key={row.name}>
+                        <input className="pf-c-check__input"
+                            name="select-checkbox-expanded-active"
+                            onChange={() => onRowToggle(row.name)}
+                            type="checkbox"
```

Comment:
Good comment. I'm working on bigger PR where I want to add additional functionality for this so it can be used for bootorder VM configuration. I will add it there.